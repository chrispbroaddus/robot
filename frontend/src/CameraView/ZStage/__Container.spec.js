import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import ZStageContainer from './__Container.js'

describe('ZStageContainer', () => {
  let props

  beforeEach(() => {
    props = {
      isOpen: true,
      onRequestClose: () => {},
      onSendValue: () => {}
    }
  })

  it('should render self and subcomponents', () => {
    const wrapper = shallow(<ZStageContainer {...props} />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
