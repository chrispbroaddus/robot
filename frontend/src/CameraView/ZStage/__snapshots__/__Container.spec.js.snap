// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ZStageContainer should render self and subcomponents 1`] = `
<styled.div>
  <Modal
    ariaHideApp={true}
    bodyOpenClassName="ReactModal__Body--open"
    className="zstage-modal"
    closeTimeoutMS={0}
    contentLabel="Z Stage"
    isOpen={true}
    onRequestClose={[Function]}
    overlayClassName="modal-overlay"
    parentSelector={[Function]}
    portalClassName="ReactModalPortal"
    shouldCloseOnOverlayClick={true}
    shouldFocusAfterRender={true}
  >
    <styled.div>
      <Styled(MdClose)
        onClick={[Function]}
      />
      <styled.h5>
        Z-Stage
      </styled.h5>
      <styled.div>
        <Styled(Leg) />
        <styled.div
          innerRef={[Function]}
          onMouseDown={[Function]}
          onMouseUp={[Function]}
        >
          <Body
            innerRef={[Function]}
            onMouseMove={[Function]}
          />
        </styled.div>
        <Styled(Leg) />
      </styled.div>
      <styled.div>
        <span>
          5.5
        </span>
         in
      </styled.div>
      <styled.div>
        Drag center of robot to position
      </styled.div>
    </styled.div>
  </Modal>
</styled.div>
`;
